### PR TITLE
feat: memo structure in xero app

### PR DESCRIPTION
**Pull request by NileshPant1999**
### Description
memo structure in xero app

## Clickup
https://app.clickup.com/tsdfsdfsd/

## Release Notes

- **New Features**
	- Enhanced customization options for exporting data to Xero, including new fields for memo structures.
	- Added a "Customization" section in the settings interface, allowing users to select data points for line item-level descriptions.
	- Introduced a preview area for users to see the current memo text.

- **Improvements**
	- Improved clarity and guidance for users on configuring export settings.
	- Enhanced user experience with dynamic updates to memo previews based on selected options.

### CODE REVIEW

File: src/app/branding/c1-contents-config.ts [hunk 1]
```diff
@@ -214,6 +214,8 @@ export const c1Contents = {
                     cccExpenseBankAccountLabel: 'Which bank account should the bank transactions post to?',
                     cccExpenseStateSubLabel: 'You can choose to only export expenses when they\'ve been labeled approved or closed. '
                 },
+                memoStructureLabel: 'Set the line-item description field in NetSuite',
```
Comment (NileshPant1999) [inline src/app/branding/c1-contents-config.ts:217]: Xero

File: src/app/branding/fyle-contents-config.ts [hunk 2]
```diff
@@ -246,6 +246,10 @@ export const fyleContents = {
                 billPaymentAccountSubLabel: ', the payment entries will be posted to the selected Payment account in ',
                 postEntriesCurrentPeriod: 'Post entries in the current accounting period',
                 contentText: 'In this section, you can customize the integration based on your accounting requirements. ',
+                topLevelMemoStructureLabel: 'Customize the Top-Level Memo Field',
+                topLevelMemoStructureSubLabel: 'Select the datapoints you\'d like to export to QuickBooks Desktop’s top-level memo field when exporting expenses from Fyle.',
```
Comment (NileshPant1999) [inline src/app/branding/fyle-contents-config.ts:250]: Xero

File: src/app/branding/c1-contents-config.ts (hunk 1 above)
Comment (anishfyle) [reply src/app/branding/c1-contents-config.ts:217]: instead of netsuite, to make it xero

Comment (NileshPant1999) [reply src/app/branding/c1-contents-config.ts:217]: maine already daal rakha tha 😆

**Review by anishfyle — APPROVED** [review]

**Review by DhaaraniCIT — APPROVED** [review]

**Pull request merged**